extensions/amp-minute-media-player/amp-minute-media-player.md

```diff
@@ -41,7 +43,7 @@ Example with RESPONSIVE layout - width and height attributes determine the aspec
 </amp-minute-media-player>
 ```
 
-Example with FIXED layout - fixed width and height.
+Example with `fixed` layout - fixed width and height.
```

Comment:
would it make sense to link to [something link this](https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout#supported-values-for-the-layout-attribute) here?